Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -570,9 +570,14 @@ extension PurchasesOrchestrator: StoreKit1WrapperDelegate {
         let storeTransaction = StoreTransaction(sk1Transaction: transaction)
 
         switch transaction.transactionState {
-        case .restored, // for observer mode
-             .purchased:
-            self.handlePurchasedTransaction(storeTransaction, storefront: storeKit1Wrapper.currentStorefront)
+        case .restored: // for observer mode
```

Comment:
not from this PR, but we should probably expand with some comments here re: observer mode: 

When something comes through the `.restore` flow in the queue, that's actually not even from our SDK - it's from a call to [restoreCompletedTransactions](https://developer.apple.com/documentation/storekit/skpaymentqueue/1506123-restorecompletedtransactions#), which we don't use

Reply:
I'll add a comment 👍🏻